eclipse-collections-testutils/pom.xml

```diff
@@ -56,6 +56,12 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Is there any reason why you aren't moving the junit4 to to jupiter-vintage so that the dependency on junit4 is removed altogether? Doing this required no code changes afair from my PR.

Reply:
The reason that declaring a dependency on junit-vintage-engine works for you is that it depends on junit 4 and pulls it in as a transitive dependency.

https://mvnrepository.com/artifact/org.junit.vintage/junit-vintage-engine/5.10.2

This engine is used to launch JUnit 4 code, and should be used as a runtime dependency of the surefire plugin.

https://github.com/eclipse/eclipse-collections/blob/master/pom.xml#L480-L496